MatrixSDK/NotificationCenter/MXNotificationCenter.h

```diff
@@ -197,8 +197,9 @@ extern NSString *const kMXNotificationCenterAllOtherRoomMessagesRuleID;
  
  @param pushRule the push rule to update
  @param enable YES to enable.
+ @param completion an optional completion block for the operation.
  */
-- (void)enableRule:(MXPushRule*)pushRule isEnabled:(BOOL)enable;
+- (void)enableRule:(MXPushRule*)pushRule isEnabled:(BOOL)enable completion:(void (^_Nullable)(NSError * _Nullable error))completion;
```

Comment:
If I'm not mistaken adding nullability specifiers for just parts of this file will introduce a ton of new warnings for the rest of the items. You should probably have a look at `MX_ASSUME_MISSING_NULLABILITY_BEGIN`

Reply:
Oh, I didn't know.
Applied -> https://github.com/matrix-org/matrix-ios-sdk/pull/1702/commits/1f5997c8aee6cc36bccbd1b3ac1cbc37c020aca2